### PR TITLE
Clean up and localize several NPC skill variants

### PR DESCRIPTION
**Pull request by TikaelSol**
This is all of them except PFS, which are, as expected, a mess.

### CODE REVIEW

File: packs/data/blog-bestiary.db/kobold-tunnelrunner.json [hunk 1]
```diff
@@ -407,7 +407,7 @@
                         "key": "FlatModifier",
                         "label": "PF2E.SkillVariant.Traps",
                         "predicate": [
-                            "trap"
+                            "actioin:craft:trap"
```
Comment (simonward) [inline packs/data/blog-bestiary.db/kobold-tunnelrunner.json:410]: action

Comment (TikaelSol) [inline packs/data/blog-bestiary.db/kobold-tunnelrunner.json:410]:
```suggestion
                            "action:craft:trap"
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -1080,19 +1082,24 @@
             "ForestsOrJungles": "In Forests or Jungles",
             "ForestsOrSnow": "In Forests or Snow",
             "ForestsOrSwamps": "In Forests or Swamps",
+            "ForestsOrUnderground": "In Forsts or Underground Areas",
```
Comment (TikaelSol) [inline static/lang/re-en.json:1085]:
```suggestion
            "ForestsOrUnderground": "In Forests or Underground Areas",
```

**Comment by stwlam** [comment]
Conflicts!

**Comment by TikaelSol** [comment]
Fixed

**Pull request merged**